product_details/storage.py

```diff
@@ -181,3 +181,50 @@ def update(self, name, content, last_modified):
 
         with open(lm_fn, 'w') as lm_fo:
             lm_fo.write(last_modified)
+
+    def all_json_files(self):
+        json_files = []
+        for root, dirs, files in os.walk(self.json_dir):
+            root = os.path.relpath(root, self.json_dir)
+            root = '' if root == '.' else root
+            json_files.extend(os.path.join(root, f) for f in files if f.endswith('.json'))
+
+        return json_files
+
+
+def json_file_data_to_db(model=None):
+    """Import JSON file data into the DB.
+
+    This is a function to be used in a data migration.
+    It's here mostly so that it can be imported and tested.
+    """
+    PDModel = model or ProductDetailsFile
+    if PDModel.objects.count():
```

Comment:
NIT: I would prefer to use `exists()` for this situation; `count()` can be slow and the value isn't used here.